web/frontend/components/HealthSummary/HealthSummary.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+const getHealthIcon = (health) => {
+  switch (health) {
+    case 'critical':
+      return <i className="eos-icons health-summary-icon text-danger">error</i>;
+    case 'warning':
+      return (
+        <i className="eos-icons health-summary-icon text-warning">warning</i>
+      );
+    case 'passing':
+      return (
+        <i className="eos-icons health-summary-icon text-success">
+          check_circle
+        </i>
+      );
+    default:
+      return (
+        <i className="eos-icons health-summary-icon text-muted">
+          fiber_manual_record
+        </i>
+      );
+  }
+};
+
+const any = (predicate, label) =>
+  Object.keys(predicate).reduce((accumulator, key) => {
+    if (accumulator) {
+      return true;
+    }
+    return predicate[key] === label;
+  }, false);
+
+const getCounters = (data) =>
+  data.reduce(
+    (accumulator, element) => {
+      if (any(element, 'critical')) {
+        return { ...accumulator, critical: accumulator.critical + 1 };
+      }
+
+      if (any(element, 'warning')) {
+        return { ...accumulator, warning: accumulator.warning + 1 };
+      }
+
+      if (any(element, 'passing')) {
+        return { ...accumulator, passing: accumulator.passing + 1 };
+      }
+      return accumulator;
+    },
+    { critical: 0, warning: 0, passing: 0 }
+  );
+
+const HealthSummary = ({ data }) => {
+  const counters = getCounters(data);
+
+  return (
+    <div>
+      <div className="col">
+        <div className="row">
+          <div className="col">
+            <h1>At a glance</h1>
+          </div>
+        </div>
+        <hr className="margin-10px" />
+
+        <h5>Global Health</h5>
+        <div className="health-container horizontal-container">
+          <div className="alert alert-inline alert-success health-passing">
+            <i className="eos-icons-outlined eos-18 alert-icon">check_circle</i>
+            <div className="alert-body">Passing</div>
+            <span className="badge badge-secondary">{counters.passing}</span>
+          </div>
+          <div className="alert alert-inline alert-warning health-warning">
+            <i className="eos-icons-outlined eos-18 alert-icon">warning</i>
+            <div className="alert-body">Warning</div>
+            <span className="badge badge-secondary">{counters.warning}</span>
+          </div>
+          <div className="alert alert-inline alert-danger health-critical">
+            <i className="eos-icons-outlined eos-18 alert-icon">error</i>
+            <div className="alert-body">Critical</div>
+            <span className="badge badge-secondary">{counters.critical}</span>
+          </div>
+        </div>
+
+        <div className="health-summary">
+          <div className="health-summary-header">
+            <span className="health-summary-id health-summary-cell">SID</span>
+            <span className="health-summary-cell">SAP Systems</span>
```

Comment:
Did you push the commit where you changed `SAP Systems` by `SAP instances`?
And the order change.
I think that's missing